src/main/resources/db/migration/V1.15__create_table_organization.sql

```diff
@@ -8,8 +8,8 @@ create table organization
     address          varchar(255)   null,
     phone            integer        null,
     email            varchar(255)   not null,
-    welcome_text     varchar(255)   not null,
```

Comment:
Evitemos modificar scripts que ya están en main, hacer un alter de la tabla en un script nuevo por favor.

`ALTER TABLE organization MODIFY COLUMN <column> text not null;`